Exchange/SelectCurrencyView/SelectCurrencyViewController.swift

```diff
@@ -113,47 +95,40 @@ extension SelectCurrencyViewController: UITableViewDelegate {
         _ tableView: UITableView,
         didSelectRowAt indexPath: IndexPath
     ) {
-        bindings.didSelectCell(indexPath)
+        let cellViewModel = cellViewModels.value[indexPath.row]
+        didSelectCurrency.accept(cellViewModel)
     }
 }
 //MARK: - Implement ViewType
 
 extension SelectCurrencyViewController: ViewType {
     typealias ViewModel = SelectCurrencyViewModel
     
+    var bindings: ViewModel.Bindings {
+        .init(
+            didSelectCurrency: didSelectCurrency.asSignal(),
+            searchCurrency: searchController.searchBar.rx.text.asDriver()
+        )
+    }
+    
     func bind(to viewModel: ViewModel) {
-        viewModel.cellViewModels.bind { [weak self] cellsModels in
-            self?.cellViewModels = cellsModels
-            DispatchQueue.main.async {
+        cellViewModels.asDriver()
+            .drive(onNext: { [weak self] _ in
                 self?.tableView.reloadData()
-            }
-        }
+            })
+            .disposed(by: disposeBag)
         
-        viewModel.isIndicatorEnabled.bind{ [weak self] condition in
-            if condition {
-                self?.activityIndicator.startAnimating()
-            } else {
-                self?.activityIndicator.stopAnimating()
-            }
-        }
-        
-        viewModel.networkErrorInBox.bind { [weak self] error in
-            guard let self = self else { return }
-            guard error != nil else { return }
-            self.showAlert()
-        }
-    }
-}
-//MARK: - SearchResultsUpdating
+        viewModel.cellViewModels
+            .drive(onNext: { [weak self] cellModels in
+                self?.cellViewModels.accept(cellModels)
+            })
```

Comment:
Если уже разобрался, что происходит под капотом, то рекомендую привыкать к короткой записи

```suggestion
            .drive(cellViewModels)
```

Reply:
да, понимать стал вроде, исправил